src/changes/revertChange.ts

```diff
@@ -67,17 +67,18 @@ export function revertSplitNodeChange(tr: Transaction, change: IncompleteChange,
   }
 }
 
-export function revertWrapNodeChange(tr: Transaction, change: IncompleteChange) {
+export function revertWrapNodeChange(tr: Transaction, change: IncompleteChange, deleteMap: Mapping) {
   const from = tr.mapping.map(change.from)
   const to = tr.mapping.map(change.to)
   const node = tr.doc.nodeAt(from)
-  // we use replaceWith for inline node, as lift will not work with inline node
+  // we use ReplaceAroundStep for inline node, as lift will not work with inline node and will help to get right mapping
   if (node?.isInline) {
-    tr.replaceWith(from, to, node.content)
+    tr.step(new ReplaceAroundStep(from, to, from + 1, to - 1, Slice.empty, 0))
```

Comment:
are you sure about gapping +1, -1? Can you please add comment in code why its needed?

Reply:
That is to remove wrapped node, and we used ReplaceAroundStep to give us right mapping as unwrapping in ReplaceStep will be mapped as delete in tr. add comment in the code

Reply:
Cool, please add into the code. Thanks!